enterprise/server/remote_execution/containers/firecracker/firecracker_test.go

```diff
@@ -105,6 +106,10 @@ func getTestEnv(ctx context.Context, t *testing.T) *testenv.TestEnv {
 	err := os.Setenv("REGISTRY_AUTH_FILE", filepath.Join(tmp, "auth.json"))
 	require.NoError(t, err)
 
+	// Clean up any lingering networking changes from previous test runs.
+	err = networking.DeleteNetNamespaces(ctx)
```

Comment:
is this safe? I would think this would introduce a data race if multiple fc tests happen to be running concurrently on the same machine, but maybe I haven't thought it through all the way.

Reply:
The firecracker tests have to run exclusively today because of our "vmIdx" thing that starts at 0 and assigns IPs deterministically from the index. (The bare pool is configured with exclusive_task_scheduling).

Reply:
I don't think we want to add more code that depends on firecracker tests being run exclusively -- sort of going in the wrong direction you know? But I'm happy to approve this temporarily if you:
- add a TODO to remove it
- ideally also take a look in a followup at fixing the vmIdx stuff in a better way?

Reply:
adding a TODO for now - will try to make the vmIdx stuff concurrency-safe (across executor processes) in a separate PR